templates/settings.html

```diff
@@ -82,6 +84,142 @@ <h1 class="page-header">Settings</h1>
                 <label for="use_map">Show campaign results map</label>
             </div>
         </div>
+        <!-- Reporting Settings Begin -->
+        <div role="tabpanel" class="tab-pane" id="reportingSettings">
+            <form id="imapForm" >
+            <br />
+            <div class="row">
+                <div>
```

Comment:
For these divs, it's recommended to add the preferred [column class](https://getbootstrap.com/docs/3.3/css/#grid-options). You'll see this used by the other tabs, and for this case it's essentially it's there to indent the elements to the right a little bit, since the `row` gives it a negative margin.

Reply:
Done.